internal/otelcollector/config/log/agent/receivers.go

```diff
@@ -0,0 +1,94 @@
+package agent
+
+import (
+	"fmt"
+
+	"k8s.io/utils/ptr"
+)
+
+func makeReceivers() Receivers {
+	return Receivers{
+		FileLog: &FileLog{
+			Exclude: []string{
+				"/var/log/pods/kyma-system_telemetry-log-agent*/*/*.log",
+				"/var/log/pods/kyma-system_telemetry-fluent-bit*/*/*.log",
+			},
+			Include:         []string{"/var/log/pods/*/*/*.log"},
+			IncludeFileName: false,
+			IncludeFilePath: true,
+			StartAt:         "beginning",
+			Storage:         "file_storage",
+			Operators:       makeOperators(),
+		},
+	}
+}
+
+func makeOperators() []Operator {
+	return []Operator{
+		makeContainerParser(),
+		makeMoveToLogStream(),
+		makeJSONParser(),
+		makeCopyBodyToOriginal(),
+		makeMoveMessageToBody(),
+		makeSeverityParser(),
+	}
+}
+
+func makeContainerParser() Operator {
+	return Operator{
+		ID:                      "containerd-parser",
+		Type:                    "container",
+		AddMetadataFromFilePath: ptr.To(true),
+		Format:                  "containerd",
+	}
+}
+
+func makeMoveToLogStream() Operator {
+	return Operator{
+		ID:     "move-to-log-stream",
+		Type:   "move",
+		From:   "attributes.stream",
+		IfExpr: "attributes.stream != nil",
+		To:     "attributes[\"log.iostream\"]",
+	}
+}
+
+func makeJSONParser() Operator {
+	regexPattern := `^{(?:\\s*"(?:[^"\\]|\\.)*"\\s*:\\s*(?:null|true|false|\\d+|\\d*\\.\\d+|"(?:[^"\\]|\\.)*"|\\{[^{}]*\\}|\\[[^\\[\\]]*\\])\\s*,?)*\\s*}$`
```

Comment:
why do we need the double escapes in a raw string?
and please add some documentation what this regex should match

Reply:
lets try to use a simpler `detection` pattern such as `^\s*{.*}\s*$`

maybe it is even simpler to NOT use any such pattern and let there be Operator fail on every not-json line (performance tests would tell)

Reply:
removed